### DISCUSSION TITLE
Can we binding Foreground in DataGridTextColumn?

### DISCUSSION THREAD

**Discussion by IronEaterBeast** [AvaloniaUI/Avalonia#14815] [Q&A]
<DataGridTextColumn Header="Publish" Binding="{Binding IsPublish}"  Foreground="{Binding Color}"/>

It seems like it's not working. Can it only accept fixed values like "Yellow" and cannot binding?

**Comment by timunie** [comment]
You can, use a Style for DataCridCell

**Reply by IronEaterBeast** [reply]
I remember that using DataGridCell might disable sorting, not sure if I remembered correctly.

**Reply by timunie** [reply]
TemplateColumn maybe, but you can set SortPath

**Reply by IronEaterBeast** [reply]
I couldn't find the content of 'SortPath' in https://docs.avaloniaui.net/. I don't know how to use it. Is it 'SortPath="{Binding abc}"'?

**Reply by timunie** [reply]
`SortMemberPath="MyProperty"` iirc
